examples/tx_log.rs

```diff
@@ -1,45 +1,55 @@
+use edn_derive::Serialize;
 use transistor::client::Crux;
-use transistor::edn_rs::{ser_struct, Serialize};
 use transistor::types::http::Actions;
+use transistor::types::response::TxLogResponse;
 use transistor::types::CruxId;
 
-fn main() {
+#[cfg(not(feature = "async"))]
+fn tx_log() -> TxLogResponse {
     let person1 = Person {
         crux__db___id: CruxId::new("jorge-3"),
         first_name: "Michael".to_string(),
         last_name: "Jorge".to_string(),
     };
+    // edn_rs::to_string(person1) { :crux.db/id :jorge-3, :first-name \"Michael\", :last-name \"Jorge\", }
 
     let person2 = Person {
         crux__db___id: CruxId::new("manuel-1"),
         first_name: "Diego".to_string(),
         last_name: "Manuel".to_string(),
     };
-    println!("{:?}", edn_rs::to_string(person1.clone()));
-    //"{ :crux.db/id :jorge-3, :first-name \"Michael\", :last-name \"Jorge\", }"
-    println!("{:?}", edn_rs::to_string(person2.clone()));
-    //"{ :crux.db/id :manuel-1, :first-name \"Diego\", :last-name \"Manuel\", }"
+    // edn_rs::to_string(person2): { :crux.db/id :manuel-1, :first-name \"Diego\", :last-name \"Manuel\", }
 
     let actions = Actions::new().append_put(person1).append_put(person2);
+    // "[[:crux.tx/put { :crux.db/id :jorge-3, :first-name \"Michael\", :last-name \"Jorge\", }],
+    //   [:crux.tx/put { :crux.db/id :manuel-1, :first-name \"Diego\", :last-name \"Manuel\", }]]"
 
     let body = Crux::new("localhost", "3000")
         .http_client()
         .tx_log(actions)
         .unwrap();
-    // Request body for Actions
-    // "[[:crux.tx/put { :crux.db/id :jorge-3, :first-name \"Michael\", :last-name \"Jorge\", }],
-    //   [:crux.tx/put { :crux.db/id :manuel-1, :first-name \"Diego\", :last-name \"Manuel\", }]]"
 
-    println!("\n Body = {:?}", body);
+    return body;
+}
+
+#[test]
+#[cfg(not(feature = "async"))]
+fn test_tx_log() {
+    let tx_log = tx_log();
+    assert!(tx_log.tx___tx_id > 0)
```

Comment:
id > 0

Reply:
any test works here, because it is mostly to detect if there is no inconsistencies with date, not really going to change to is_empty sorry